src/Data/ProgrammesDb/Entity/RefOptions.php

```diff
@@ -23,147 +28,120 @@ class RefOptions
     private $id;
 
     /**
-     * @var array
+     * @var string
      *
-     * @ORM\Column(type="string", nullable=false)
+     * @ORM\Column(type="string", length=50, nullable=false)
      */
-    private $originalId;
+    private $guid;
 
     /**
-     * @var CoreEntity
+     * @var string
      *
-     * @ORM\ManyToOne(targetEntity="CoreEntity")
-     * @ORM\JoinColumn(nullable=true, onDelete="SET NULL")
+     * @ORM\Column(type="string", length=50, nullable=false)
      */
-    private $optionsForCoreEntity;
+    private $projectId;
 
     /**
-     * @var Network
+     * @var string
      *
-     * @ORM\ManyToOne(targetEntity="Network")
-     * @ORM\JoinColumn(nullable=true, onDelete="SET NULL")
+     * @ORM\Column(type="string", nullable=true)
      */
-    private $optionsForNetwork;
+    private $entity;
 
     /**
-     * @var array
+     * @var string
      *
-     * @ORM\Column(type="json_array", nullable=true)
+     * @ORM\Column(type="string", nullable=false)
```

Comment:
Add a length so the row size is shorter, it's going to be either 'admin' or 'local' so 5 chars?